instrumentation/grizzly-http-2.3.20/grizzly-http-2.3.20.gradle

```diff
@@ -0,0 +1,36 @@
+apply from: "${rootDir}/gradle/instrumentation.gradle"
+apply plugin: 'org.unbroken-dome.test-sets'
+
```

Comment:
Shouldn’t instrumentation be called `grizzly-http-2.3`?

Reply:
yes! it took a little work (https://github.com/open-telemetry/opentelemetry-auto-instr-java/pull/433/commits/18fdac626df3d95419a9c7f221e72771805169e7), but worth it to keep the version support story simple